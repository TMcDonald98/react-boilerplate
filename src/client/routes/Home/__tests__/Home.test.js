import * as React from 'react';
import {
  Home,
} from '../Home';

describe('components/Home', () => {
  describe('render', () => {
    it('should render without error', () => {
      const result = shallow(<Home />);

      expect(result).toMatchSnapshot('Home');
    });
  });
});
