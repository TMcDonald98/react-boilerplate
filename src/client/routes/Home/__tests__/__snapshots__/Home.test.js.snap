// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/Home render should render without error: Home 1`] = `
<div
  className="home-container"
>
  <Welcome />
</div>
`;
